src/internal_events/mod.rs

```diff
@@ -61,6 +62,7 @@ pub use self::blackhole::*;
 pub use self::docker::*;
 pub use self::elasticsearch::*;
 pub use self::file::*;
+pub use self::generator::*;
```

Comment:
Should guard these with `#[cfg(feature = "sources-generator")]` and then you can use `pub(crate)` too.